Vagrantfile

```diff
@@ -17,6 +17,80 @@ gopath_folder="/opt/gopath"
 
 cluster_ip_nodes = ""
 
+SUBNET_PREFIX          = "192.168."
```

Comment:
I see similar code in this PR : #613
Are those related or just a duplicate code ?